### ISSUE TITLE
Train on ImageNet?

### ISSUE THREAD

**Issue by fzyzcjy** [Bisonai/mobilenetv3-tensorflow#2]
Hi! Thanks for providing the code! I wonder did you pretrain it on ImageNet and could you please share it?

**Comment by martinkersner** [comment]
Hi @fzyzcjy!

We are planning to share pretrained weights, but currently, we do not have them.

Cheers,
Martin

**Comment by fzyzcjy** [comment]
Thanks very much! Waiting :)

**Comment by yangxue0827** [comment]
Looking forward to the weight of ImageNet. @martinkersner

**Comment by VincentChong123** [comment]
Hi @martinkersner!

If possible, besides the weights, would you please share the loss curve and param setting?
Such curve for example the one below would be helpful to monitor our training
https://github.com/PengBoXiangShang/MobileNetV3_PyTorch/raw/master/figures/training_loss.png
